llvm/test/CodeGen/AIE/aie2/ra/tie-subregs-flow-tmp.mir

```diff
@@ -0,0 +1,112 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
```

Comment:
Is this file meant to be committed? What's "tmp" about this?

Reply:
Not meant to be committed...